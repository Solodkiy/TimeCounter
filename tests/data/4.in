14.01-14.05
                     12,42  -  12.48


                23.24 - 01.13