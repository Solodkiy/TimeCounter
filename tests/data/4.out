14:01 - 14:05 = 0:04
12:42 - 12:48 = 0:06
23:24 - 01:13 = 1:49
                ----
                1:59